CMakeLists.txt

```diff
@@ -11,15 +11,48 @@ set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 include_directories (inc)
 include_directories (lib)
 
-macro (declare_mod MODNAME)
-  file (GLOB MOD_${MODNAME}_INC "inc/${MODNAME}/*")
-  file (GLOB MOD_${MODNAME}_SRC "src/${MODNAME}/*")
-  set (MOD_${MODNAME}_ALL ${MOD_${MODNAME}_INC} ${MOD_${MODNAME}_SRC})
-  add_library (mod_${MODNAME} ${MOD_${MODNAME}_ALL})
-endmacro (declare_mod)
-
 # Application modules
-declare_mod (vcf)
+set (MOD_VCF_SOURCES
```

Comment:
What I mean is grouping the explicit list, first with the files inside `inc` and then `src`. Right now they are all mixed.